staging/src/k8s.io/apiserver/pkg/server/egressselector/config_test.go

```diff
@@ -527,7 +527,7 @@ func TestValidateEgressSelectorConfiguration(t *testing.T) {
 						},
 					},
 					{
-						Name: "master",
+						Name: "controlplane",
```

Comment:
This is dup test case.

Reply:
Duplicate error is like
`Calling ValidateEgressSelectorConfiguration expected error, got [service.connection[1]: Duplicate value: "controlplane"]--- PASS: TestValidateEgressSelectorConfiguration (0.00s)`